dotcom-rendering/src/web/components/FixedMediumFastXI.tsx

```diff
@@ -0,0 +1,128 @@
+import type { DCRContainerPalette } from '../../types/front';
+import type { TrailType } from '../../types/trails';
+import {
+	Card100,
+	Card50_Card25_Card25,
+	Card66_Card33,
+} from '../lib/dynamicSlices';
+import { LI } from './Card/components/LI';
+import { UL } from './Card/components/UL';
+import { FrontCard } from './FrontCard';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+};
+
+const decideOffset = ({
```

Comment:
Non-blocking naming suggestion: Since this returns a boolean would it make more sense to name this something along the lines of `hasOffset` or `shouldHaveOffset`?